bootstrap/bootstrap.go

```diff
@@ -335,9 +335,8 @@ func (b *Bootstrap) executeHook(ctx context.Context, scope string, name string,
 
 	b.shell.Headerf("Running %s hook", name)
 
-	if redactor := b.setupRedactor(); redactor != nil {
-		defer redactor.Flush()
-	}
+	redactors := b.setupRedactor()
+	defer redactors.Flush()
```

Comment:
`redactors` is a `type RedactorMux []*Redactor` which might be `nil`, but can still safely call `Flush()`.

Reply:
:exploding_head: